docker-compose.yml

```diff
@@ -575,6 +575,8 @@ services:
       - collections-publisher
       - collections
       - publisher
+      - frontend
+      - draft-frontend
```

Comment:
Maybe just add that publishing-e2e-tests needs these, may also make sense to rebase this amongst other changes to app dependencies.